notebooks/Explore_MNTOHA_temperature_observations.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
If you are familiar with rmarkdown, but jupytext provides roughly the same functionality as rmarkdown (edit in plain text and then it renders it to a notebook).

Reply:
Thanks! This looks really neat. It looks like others would need Jupytext installed to easily read and run the notebooks. I'll check with our team to see what they think.

Reply:
That seems like a neat tool, particularly if we can make use of the [paired notebook](https://github.com/mwouts/jupytext/blob/main/README.md?plain=1#L87-L132) approach, so that the text version is update whenever changes are made in the notebook (or vice versa). Whenever you've got time for a little side project, it could be worth doing a little test case where you make a notebook Andy, commit the text version, and I test how well it works to render it, make edits, and commit those edits on my end.